cargo-workspaces/src/publish.rs

```diff
@@ -46,6 +46,7 @@ pub struct Publish {
 
 impl Publish {
     pub fn run(mut self, metadata: Metadata) -> Result {
+        let mut command_success = true;
```

Comment:
Since we no longer bail on errrors, I thought that it's weird to print `success ok` if we've met problems.